### PR TITLE
F cleanup indigo

### PR DESCRIPTION
**Pull request by ugocupcic**

### CODE REVIEW

File: sr_hand/src/sr_subscriber.cpp [hunk 1]
```diff
@@ -98,6 +107,12 @@ void SRSubscriber::sendupdateCallback( const sr_robot_msgs::sendupdateConstPtr&
 
 }
 
+void SRSubscriber::cmd_callback( const std_msgs::Float64ConstPtr& msg, std::string joint_name )
```
Comment (shadowmanos) [inline sr_hand/src/sr_subscriber.cpp]: It seems that in every C++ function of sr_hand that needs a string we are passing a copy of the string. That is unnecessarily innefficient. The C++ idiomatic/efficient function prototype would be :
void SRSubscriber::cmd_callback( const std_msgs::Float64ConstPtr& msg, const std::string &joint_name )
We should update that when we have time. Not necessarily in this pull request.

**Comment by shadowmanos** [comment]
We should probably also delete rviz_config folder as part of clean_up. Rviz has deprecated this file type and currently uses *.rviz files for configuration. We have one : 
sr_hand/config/hand_demo_view.rviz

**Comment by ugocupcic** [comment]
deleted the rviz config

File: sr_hand/src/sr_subscriber.cpp (hunk 1 above)
Comment (ugocupcic) [inline sr_hand/src/sr_subscriber.cpp]: Fixed

**Comment by shadowmanos** [comment]
What I meant was that the sr_ros_interface/rviz_config/ contains a deprecated file.
sr_ros_interface/sr_hand/config/ contains the modern file.
Unless you have other reasons to get rid of the latter ;-)

**Comment by ugocupcic** [comment]
sorry! can't see rviz_config though. Must have been deleted for indigo already.

**Comment by shadowmanos** [comment]
Yes it is! Actually I deleted it and forgot I did ;-)

**Pull request merged**